sdk/databox/arm-databox/package.json

```diff
@@ -109,5 +99,45 @@
     ],
     "disableDocsMs": true,
     "apiRefLink": "https://docs.microsoft.com/javascript/api/@azure/arm-databox?view=azure-node-preview"
+  },
+  "type": "module",
+  "tshy": {
+    "project": "./tsconfig.src.json",
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
```

Comment:
Are we shipping for React-Native and Browser support?  If not, please remove the `esmDialects`.

Reply:
@mpodwysocki do you see any potential risk of adding this support? When we deprecated the very older version of Node SDK, we have this deprecation message "This package is deprecated in favor of @azure/arm-compute whick works both on node.js and browsers" which are actually from Ramya at that time, but we do have been ignoring the browser test for mgmt plane https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/databox/arm-databox/package.json#L80 so I was not sure if we want this or not

Perhapbs @xirzec has some thoughts here ?

Reply:
After some offline discussion with @mpodwysocki, we are fine with keeping this. Going to merge this for now. @xirzec feel free to let me know if you have any concerns, we can always revisit this :)